File: cpp/trabajo2.cpp
#include <iostream>
#include <string>
using namespace std;

// Definir un tipo de dato compuesto (struct o clase)
struct Persona {
    string nombre;
    int edad;
};

int main() {
    // Crear un array de objetos Persona
    Persona personas[3];

    // Inicializar los elementos del array
    personas[0] = {"Ana", 25};
    personas[1] = {"Luis", 30};
    personas[2] = {"Marta", 22};

    // Mostrar los datos
    for (int i = 0; i < 3; i++) {
        cout << "Nombre: " << personas[i].nombre << ", Edad: " << personas[i].edad << endl;
    }

    return 0;

File: c#/trabajo2.cs
using System;

// Definir un tipo de dato compuesto (clase)
class Persona
{
    public string Nombre { get; set; }
    public int Edad { get; set; }
}

class Program
{
    static void Main(string[] args)
    {
        // Declarar un array de objetos Persona
        Persona[] personas = new Persona[3];

        // Inicializar los elementos del array
        personas[0] = new Persona { Nombre = "Ana", Edad = 25 };
        personas[1] = new Persona { Nombre = "Luis", Edad = 30 };
        personas[2] = new Persona { Nombre = "Marta", Edad = 22 };

        // Mostrar los datos
        for (int i = 0; i < personas.Length; i++)
        {
            Console.WriteLine($"Nombre: {personas[i].Nombre}, Edad: {personas[i].Edad}");
        }
    }
}